docs/README.Ubuntu.md

```diff
@@ -147,6 +148,15 @@ sudo apt install doxygen libcap-dev libsndio-dev libmariadbd-dev
 
 **[back to top](#table-of-contents)**
 
+### 3.3 Ubuntu <= 18.04
+
+Ubuntu 18.04 and lower ship an outdated [Meson](https://mesonbuild.com/) package. In order to compile Kodi with AV1 support, you have to manually update Meson by running
+```
+pip3 install meson 
```

Comment:
@Montellese You are executing this as `sudo` according to your comment. I guess I should prefix this with `sudo` then?

I'm not sure if `sudo apt-get remove meson` is really needed...

Reply:
I'm not sure either. I first ran `sudo apt-get install meson` and that didn't work. Then I ran `pip3 install meson` but `which meson` still pointed to `/usr/bin/meson` installed with `apt-get`. I then ran `sudo apt-get remove meson` but still no luck. I then tried again with `sudo pip3 install meson` but still no luck but then using `update-alternatives` solved the issue. But it might well be that I just messed up my system by first running `sudo apt-get install meson`.

Reply:
i tried again. Using `pip3 install meson` results in meson being installed to `~/.local/bin/meson`. Using `sudo pip3 install meson` results in meson being installed to `/usr/local/bin/meson`.

Reply:
Using pip3 as sudo is not recommend.
Not sure who's responsible for the entry but on my system:
`~/.local/bin` is part of my PATH
So I assume it would work for me.

Reply:
@a1rwulf Ah that's not the case on my system which is why it didn't work. Yeah I know that `sudo pip3` is not recommended.

Reply:
I wonder how that entry came on my system.
Possibly during pip3 installation?
I never thought about it...

Reply:
Thanks for the info, I added `sudo`. I also checked this in my Ubuntu 18.04 VM.

(The official installation docs don't mention `sudo`, which is a little bit confusing: https://mesonbuild.com/Quick-guide.html#installation-using-package-manager)

Reply:
> Using pip3 as sudo is not recommended.
> Not sure who's responsible for the entry but on my system:
> `~/.local/bin` is part of my PATH
> So I assume it would work for me.

Maybe we should just write to run the command as `root`?

Reply:
I'd rather encourage people to run `pip3 install meson` and make sure they add `/home/youruser/.local/bin` to the `PATH` environment variable.

Another question comes to my mind:
Is `pip3` a dependency you need when building kodi?
Otherwise we'd need to add pip3 installation also somewhere in the docs.

Reply:
> Maybe we should just write to run the command as root?

pip3 is most certainly used by a bunch of python devs, usually they never ever execute pip3 as root or sudo as they don't want to fuckup their system by chaning python stuff globally.
So running this as root is not what we want.

Reply:
> Is `pip3` a dependency you need when building kodi?

Yep, it is. It's also listed as a dependency in the "manual install" section (`python3-pip`).

Reply:
@a1rwulf I agree. I updated the commit (please review).

Relating discussion to `I wonder how that entry came on my system`: https://github.com/pypa/pip/issues/3813